tests/linux_dmabuf_v1.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2025 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "in_process_server.h"
+#include "linux_dmabuf_v1.h"
+#include "version_specifier.h"
+
+#include <gmock/gmock.h>
+
+using namespace testing;
+using namespace wlcs;
+
+class LinuxDmabufTest
+    : public wlcs::StartedInProcessServer
+{
+};
+
+TEST_F(LinuxDmabufTest, default_feedback)
+{
+    wlcs::Client client{the_server()};
+
+    auto linux_dmabuf = client.bind_if_supported<zwp_linux_dmabuf_v1>(AnyVersion);
```

Comment:
This should be 
```suggestion
    auto linux_dmabuf = client.bind_if_supported<zwp_linux_dmabuf_v1>(AtLeastVersion{4});
```
as the feedback interfaces only appear in V4 and above.